stories/ar/29.json

```diff
@@ -0,0 +1,24 @@
+{
+  "language": "arabic",
+  "story": 29,
+  "title": "قرارك",
+  "sections": [
+    "من خلال هذه القصص، وقلت لكم عن الكثير من الناس الذين يعتقدون في الله. قلت لكم عن هابيل ونوح وإبراهيم وموسى وغيرهم. انهم جميعا اعترفوا بأنهم كانوا مذنبين. انهم يعتقدون انهم يستحقون الموت.",
+    "كما أعرب عن ثقته في المخلص الذي قال الله سيأتي. لذلك، غفر الله لهم خطاياهم وخلصهم من العقاب الأبدي.",
+    "كان هناك، وسوف يكون دائما طريقة واحدة فقط ليتم حفظها من الخطيئة، وفصل من الله والعقوبة الأبدية، عليك أن تؤمن مثل هابيل ونوح وإبراهيم وموسى أنك آثم الذي يستحق عقوبة الموت. عليك أن تؤمن المخلص، يسوع، الذي توفي بدلا منك، لذلك يمكن مغفورة لك خطاياك.",
+    "البعض منكم قد يعتقد، \"وكانت هذه قصة جيدة، ولكن أريد أن تفكر في ذلك وقررت لاحقا إذا أنا ذاهب إلى الاعتقاد يسوع هو منقذي.\" أريدك أن الاستماع إلى القصة وانا ذاهب الآن أن أقول لك .",
+    "يوم واحد، عندما كان يسوع ما زال هنا على الأرض، وقال الناس هذه القصة حول بحيرة النار. وقال: \"كان هناك رجل غني الذين يرتدون ملابس جميلة جدا. عاش في ترف كل يوم، ولكن عند بابه وضعت متسول اسمه لعازر.",
```

Comment:
not seeing what you're seeing, but i can tell you that the file is valid json and i can page through all the sections of the 29th story in arabic